lives-of-primitive-quakers/modernized/08-chapter-7.adoc

```diff
@@ -795,19 +802,19 @@ to the end that such course may be taken with them here,
 as shall be agreeable to our laws, and their demerits.
 And for so doing, these our letters shall be your sufficient warrant and discharge.
 
-"`Given at our court at Whitehall, the 9th day of September, 1661,
-in the thirteenth year of our reign.
+[.signed-section-closing]
+"`Given at our court at Whitehall, the 9th day of September, 1661, in the thirteenth year of our reign.
 
+[.signed-section-closing]
 "`By his Majesty's command,
 
+[.signed-section-signature]
 "`William Morris.
 
-Addressed: "`To our trusty and well-beloved John Endicot, Esq.,
-and to all and every other the governor, or governors of our plantations of New England,
-and of all the colonies thereunto belonging, that now are, or hereafter shall be;
-and to all and every the ministers and officers
-of our said plantations and colonies whatsoever,
-within the continent of New England.`"
+[.postscript]
+"`Addressed: To our trusty and well-beloved John Endicot, Esq., and to all and every other the governor, or governors of our plantations of New England, and of all the colonies thereunto belonging, that now are, or hereafter shall be; and to all and every the ministers and officers of our said plantations and colonies whatsoever, within the continent of New England.`"
```

Comment:
`[.postscript]` paragraphs don't need to be all on one line like a heading, so this should probably be switched back to how it was in terms of lines.